src/commands/service/pipeline.ts

```diff
@@ -189,6 +192,22 @@ export const execute = async (
     }
 
     await fetchValues(serviceName, opts);
+    const bedrockFile: BedrockFile = loadBedrockFile(projectPath);
+    let servicePath = "";
+    bedrockFile.services.forEach((service) => {
+      if (service.displayName === serviceName) {
+        servicePath = service.path;
+        return;
+      }
+    });
+
+    if (servicePath === "") {
+      throw buildError(errorStatusCode.VALIDATION_ERR, {
+        errorKey: "project-pipeline-err-service-missing",
+        values: [serviceName],
```

Comment:
Do we have unit test for this?

Reply:
@edaena 
<img width="1022" alt="image" src="https://user-images.githubusercontent.com/3119513/81240684-ee213a00-8fbc-11ea-8a6c-1df2a59a879a.png">

Reply:
No, I'll make the update.